Modules/Loadable/Markups/MRML/vtkMRMLMarkupsROINode.h

```diff
@@ -168,15 +164,24 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsROINode : public vtkM
   /// Create default storage node or nullptr if does not have one
   vtkMRMLStorageNode* CreateDefaultStorageNode() override;
 
-  /// Create default storage node or nullptr if does not have one
+  /// Create default display node or nullptr if does not have one
   void CreateDefaultDisplayNodes() override;
 
+  /// Reimplemented to recalculate the axis-aligned bounds of the ROI.
+  /// If the ROI is rotated, this function will not reflect the oriented bounds defined by the ROI.
+  /// To get the planes that define the oriented bounding box, use GetPlanes()/GetPlanesWorld().
+  /// \sa GetPlanes(), GetPlanesWorld()
```

Comment:
Could you describe the difference between the two methods?

Reply:
Ok, done!